.simplecov

```diff
@@ -0,0 +1,8 @@
+# frozen_string_literal: true
+
+SimpleCov.start do
+  enable_coverage :branch
+  minimum_coverage line: 99.13, branch: 90.63
```

Comment:
It was set based on the current coverage.
<img width="2033" src="https://user-images.githubusercontent.com/13041216/210383638-4824aeae-c3e3-4acd-ae95-914cb93a09a4.png">